cms/djangoapps/contentstore/tests/tests.py

```diff
@@ -85,6 +87,36 @@ def activate_user(self, email):
         self.assertTrue(user(email).is_active)
 
 
+@pytest.mark.django_db
+def test_create_account_email_already_exists(django_db_use_migrations):
+    """
```

Comment:
I am not super thrilled about how I had to make these tests functions, but burned several hours trying to get this working in class based tests and this was the best solution of the batch.

Reply:
This is actually the preferred testing style for pytest, since it's simpler (`unittest` only looks the way it does because it was derived from JUnit).  But you still have to inherit from `django.test.TestCase` if you want to use the spiffy transaction + savepoints layering for more efficient database setup via `setUpTestData()` (which isn't really needed for this test).